src/SFA.DAS.EmployerIncentives.Web.SystemAcceptanceTests/Features/Hub/EmployerIncentivesHub.feature

```diff
@@ -6,14 +6,24 @@ Feature: EmployerIncentivesHub
 
 Scenario: An employer with a single legal entity visits the EI hub page
 Given an employer has a single legal entity in their account
+And the employer has previously supplied their bank details
 When the employer accesses the hub page
 Then they are presented with a link to apply for the employer incentive
 And they are presented with a link to view previous applications
 And the back link goes to the manage apprenticeships page
 
 Scenario: An employer with a multiple legal entities visits the EI hub page
 Given an employer has a multiple legal entities in their account
+And the employer has previously supplied their bank details
 When the employer accesses the hub page
 Then they are presented with a link to apply for the employer incentive
 And they are presented with a link to view previous applications
-And the back link goes to the choose organisation page
\ No newline at end of file
+And the back link goes to the choose organisation page
+
+Scenario: An employer with no submitted bank details visits the EI hub page
+Given an employer has a single legal entity in their account
+And the employer has not yet supplied bank details
+When the employer accesses the hub page
+Then they are presented with a link to apply for the employer incentive
```

Comment:
Be better if this didn't include the solution but just the requirement.  So instead of

"they are presented with a link to apply for the employer incentive"

it would be something like...

"they can apply for the employer incentive" 

and 

"they can view previous applications" 

 etc...